config/sync/webform.webform.contact.yml

```diff
@@ -155,7 +155,7 @@ settings:
   confirmation_type: message
   confirmation_url: '<front>'
   confirmation_title: ''
-  confirmation_message: '<p>Din besked er blevet sendt.</p>'
+  confirmation_message: '<p>Din forespørgsel er blevet modtaget. Du vil modtage en bekræftelse på mail.</p>'
```

Comment:
I think English should be the standard and Danish translations is the job of DDF to manage

Reply:
I think you are somewhat correct. I myself wasn't sure how to handle this.
The reason I ended up doing it like this is because DDF going forward are supposed to be able to create their own webforms and probably are gonna write these things in danish instead of first writing them in english and then translating.
But I'll change it to english for now.